### PR TITLE
Travis Eslint and Prettier

### PR DESCRIPTION
**Pull request by NBNARADHYA**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - lts/*
+
+jobs:
+  include:
+    - stage: 'Test the server'
+      before_install: cd backend
+      script: npm run lint
```
Comment (nveenjain) [inline .travis.yml]: Won't npm install be required?

File: backend/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "globals": {
+    "Atomics": "readonly",
+    "SharedArrayBuffer": "readonly"
+  },
+  "parserOptions": {
+    "ecmaVersion": 2018
+  },
+  "plugins": ["prettier"],
+  "rules": {
+    "indent": ["off", "tab"],
```
Comment (nveenjain) [inline backend/.eslintrc.json]: indent should be an error...

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+    - stage: 'Test frontend code'
+      before_install: cd frontend
```
Comment (nveenjain) [inline .travis.yml]: create an empty folder named fronted then...

**Comment by NBNARADHYA** [comment]
npm ci will be executed by default...Travis does that

File: .travis.yml (hunk 1 above)
Comment (NBNARADHYA) [reply .travis.yml]: npm ci will be executed by default...Travis does that

**Pull request merged**